custom_components/tuya_local/devices/DH-CSK03W_dehumidifier.yaml

```diff
@@ -0,0 +1,95 @@
+name: "DH-CSK03W Dehumidifier"
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 2
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 40
+        max: 70
+      mapping:
+        - step: 5
+    - id: 5
+      name: mode
+      type: string
+      mapping:
+        - dps_val: "mode_strong"
+          value: "Strong"
```

Comment:
It is generally better to use predefined modes with [translations](https://github.com/home-assistant/core/blob/dev/homeassistant/components/humidifier/strings.json#L62), such as `boost` and `sleep` (all lower case)

Reply:
Using `boost` now and corrected `sleep` to lower case, displaying correctly in HA